android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/DimmedImageBackground.kt

```diff
@@ -0,0 +1,48 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color.Companion.Black
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.ui.theme.Black20
+
+@Composable
+fun DimmedImageBackground(@DrawableRes imageRes: Int) {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Black)
+    ) {
+        Box(modifier = Modifier.fillMaxSize()) {
+            Image(
+                painter = painterResource(id = imageRes),
+                contentDescription = null,
```

Comment:
@luongvo Can we ignore this property? 🤔

Reply:
@doannimble Can't, it's a required field

![image](https://user-images.githubusercontent.com/16315358/208351360-b0100735-de9d-4f82-8dac-c4c5d0963429.png)

Reply:
@luongvo Oh, how funny it is, nullable but required 😄 . Agree with the current approach 👌